src/server/streamableHttp.ts

```diff
@@ -64,12 +91,13 @@ export class StreamableHTTPServerTransport implements Transport {
   // when sessionId is not set (undefined), it means the transport is in stateless mode
   private sessionIdGenerator: () => string | undefined;
   private _started: boolean = false;
-  private _responseMapping: Map<RequestId, ServerResponse> = new Map();
+  private _streamMapping: Map<string, ServerResponse> = new Map();
+  private _requestToStreamMapping: Map<RequestId, string> = new Map();
   private _requestResponseMap: Map<RequestId, JSONRPCMessage> = new Map();
   private _initialized: boolean = false;
   private _enableJsonResponse: boolean = false;
-  private _standaloneSSE: ServerResponse | undefined;
-
+  private _standaloneSseStreamId: string = 'standalonesse';
```

Comment:
Nit: pick a more "special" looking stream ID?
```suggestion
  private _standaloneSseStreamId: string = '_GET_stream';
```